CHANGES.txt

```diff
@@ -73,6 +73,8 @@
     JSON serializations with UTF-8 or other non-ASCII encodings.
   * Added experimental support for directly assigning numpy scalars and array.
   * Improve the calculation of public_dependencies in DescriptorPool.
+  * Disallow setting fields to numpy singleton arrays or repeated fields to numpy
+    multi-dimensional arrays.
```

Comment:
can you call out more clearly that this is a breaking change and what the solution is?

Reply:
Done. Also fixed in release notes.